packages/polkadot/src/__snapshots__/assetHubPolkadot.multisig.e2e.test.ts.snap

```diff
@@ -1,6 +1,8 @@
 // Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
 
-exports[`AssetHubPolkadot Multisig > a redundant approval fails > events when redundant approval fails 1`] = `
+exports[`AssetHubPolkadot Multisig > approval with max weight too low fails > events when Bob executes multisig operation with low weight 1`] = `[]`;
```

Comment:
is empty event intended? I am considering add assertion to disallow empty array in snapshot because that usually means something isn't correct. if it truly should be empty, then just assert length === 0 instead of match snapshot